samtranslator/model/sam_resources.py

```diff
@@ -66,7 +66,7 @@ class SamFunction(SamResourceMacro):
         "VpcConfig": PropertyType(False, is_type(dict)),
         "Role": PropertyType(False, is_str()),
         "AssumeRolePolicyDocument": PropertyType(False, is_type(dict)),
-        "Policies": PropertyType(False, one_of(is_str(), list_of(one_of(is_str(), is_type(dict), is_type(dict))))),
+        "Policies": PropertyType(False, one_of(is_str(), is_type(dict), list_of(one_of(is_str(), is_type(dict))))),
```

Comment:
Nit: Is this just a reorder? It is hard to tell from the diff.

Reply:
There was a typo inside `list_of(one_of(is_str(), is_type(dict), is_type(dict)))`, two repeated `is_type(dict)` is meaningless. I checked [this property](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-policies) in `AWS::Serverless::Function`, the acceptable type is `String | List | Map`, so this property should be one of the tree types: `is_str()`, `is_type(dict)`, or list of `one_of(is_str(), is_type(dict))`.